api/v1/storagecluster_types.go

```diff
@@ -211,6 +211,11 @@ type ManageCephCluster struct {
 
 	// Ceph Config options
 	CephConfig map[string]map[string]string `json:"cephConfig,omitempty"`
+
+	// WipeDevicesFromOtherClusters wipes the OSD disks belonging to other clusters. This is useful in scenarios where storage cluster
+	// was reinstalled but OSD disk still contains the metadata from previous storage cluster.
+	// +optional
+	WipeDevicesFromOtherClusters bool `json:"wipeDevicesFromOtherClusters,omitempty"`
```

Comment:
This setting would be better to keep under a subsection CleanUpPolicy, As the CleanUpPolicy section contains few other values as well. A better thing would be don't expose just this one field instead Put the CleanUpPolicy spec here and pass it directly to the cephCluster if defined. This would help in future expansion.